01/23/2025 19:13:53 - INFO - __main__ - ***** Running training *****
01/23/2025 19:13:53 - INFO - __main__ -   Num examples = 90092
01/23/2025 19:13:53 - INFO - __main__ -   Num batches each epoch = 704
01/23/2025 19:13:53 - INFO - __main__ -   Num Epochs = 114
01/23/2025 19:13:53 - INFO - __main__ -   Instantaneous batch size per device = 64
01/23/2025 19:13:53 - INFO - __main__ -   Total train batch size (w. parallel, distributed & accumulation) = 128
01/23/2025 19:13:53 - INFO - __main__ -   Gradient Accumulation steps = 1
01/23/2025 19:13:53 - INFO - __main__ -   Total optimization steps = 80000
01/23/2025 19:13:53 - INFO - __main__ -  do_classifier_free_guidance = True
01/23/2025 19:13:53 - INFO - __main__ -  conditioning_dropout_prob = 0.05
Steps:   0%|                                                                                                        | 1/80000 [00:09<214:12:44,  9.64s/it]01/23/2025 19:14:05 - INFO - __main__ - Running unseen_object_with_random_light_condition validation...
cc_projection/diffusion_pytorch_model.safetensors not found
The config attributes {'cc_projection': ['pipeline_zero1to3', 'CCProjection']} were passed to Neural_Gaffer_StableDiffusionPipeline, but are not expected and will be ignored. Please verify your model_index.json configuration file.
Keyword arguments {'cc_projection': ['pipeline_zero1to3', 'CCProjection']} are not expected by Neural_Gaffer_StableDiffusionPipeline and will be ignored.
114it [07:59,  4.21s/it]
01/23/2025 19:22:55 - WARNING - root - Only 108 Image will be uploaded.
01/23/2025 19:23:03 - INFO - __main__ - Running unseen_object_with_unseen_envir validation...
cc_projection/diffusion_pytorch_model.safetensors not found
The config attributes {'cc_projection': ['pipeline_zero1to3', 'CCProjection']} were passed to Neural_Gaffer_StableDiffusionPipeline, but are not expected and will be ignored. Please verify your model_index.json configuration file.
Keyword arguments {'cc_projection': ['pipeline_zero1to3', 'CCProjection']} are not expected by Neural_Gaffer_StableDiffusionPipeline and will be ignored.
114it [07:59,  4.21s/it]
01/23/2025 19:31:53 - WARNING - root - Only 108 Image will be uploaded.
Traceback (most recent call last):
  File "/share/phoenix/nfs02/S6/localdisk/hj453/code/Neural_Gaffer/neural_gaffer_training.py", line 1267, in <module>
  File "/share/phoenix/nfs02/S6/localdisk/hj453/code/Neural_Gaffer/neural_gaffer_training.py", line 1130, in main
  File "/share/phoenix/nfs02/S6/localdisk/hj453/code/Neural_Gaffer/neural_gaffer_training.py", line 198, in log_validation
    tracker.log({split: formatted_images}, step=cur_step)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/accelerate/tracking.py", line 81, in execute_on_main_process
    return function(self, *args, **kwargs)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/accelerate/tracking.py", line 337, in log
    self.run.log(values, step=step, **kwargs)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 451, in wrapper
    return func(self, *args, **kwargs)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 403, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 393, in wrapper
    return func(self, *args, **kwargs)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1922, in log
    self._log(data=data, step=step, commit=commit)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1641, in _log
    self._partial_history_callback(data, step, commit)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1513, in _partial_history_callback
    self._backend.interface.publish_partial_history(
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 631, in publish_partial_history
    data = history_dict_to_json(run, data, step=user_step, ignore_copy_err=True)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/utils.py", line 54, in history_dict_to_json
    payload[key] = val_to_json(
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/utils.py", line 127, in val_to_json
    return items[0].seq_to_json(items, run, key, namespace)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 547, in seq_to_json
    sizes_match = all(size_equals_image(img) for img in seq)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 547, in <genexpr>
    sizes_match = all(size_equals_image(img) for img in seq)
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 544, in size_equals_image
    img_width, img_height = image.image.size  # type: ignore
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 689, in image
    self._image.load()
  File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/PIL/ImageFile.py", line 269, in load
    n, err_code = decoder.decode(b)
KeyboardInterrupt
[rank0]: Traceback (most recent call last):
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/code/Neural_Gaffer/neural_gaffer_training.py", line 1267, in <module>
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/code/Neural_Gaffer/neural_gaffer_training.py", line 1130, in main
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/code/Neural_Gaffer/neural_gaffer_training.py", line 198, in log_validation
[rank0]:     tracker.log({split: formatted_images}, step=cur_step)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/accelerate/tracking.py", line 81, in execute_on_main_process
[rank0]:     return function(self, *args, **kwargs)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/accelerate/tracking.py", line 337, in log
[rank0]:     self.run.log(values, step=step, **kwargs)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 451, in wrapper
[rank0]:     return func(self, *args, **kwargs)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 403, in wrapper_fn
[rank0]:     return func(self, *args, **kwargs)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 393, in wrapper
[rank0]:     return func(self, *args, **kwargs)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1922, in log
[rank0]:     self._log(data=data, step=step, commit=commit)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1641, in _log
[rank0]:     self._partial_history_callback(data, step, commit)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/wandb_run.py", line 1513, in _partial_history_callback
[rank0]:     self._backend.interface.publish_partial_history(
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/interface/interface.py", line 631, in publish_partial_history
[rank0]:     data = history_dict_to_json(run, data, step=user_step, ignore_copy_err=True)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/utils.py", line 54, in history_dict_to_json
[rank0]:     payload[key] = val_to_json(
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/utils.py", line 127, in val_to_json
[rank0]:     return items[0].seq_to_json(items, run, key, namespace)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 547, in seq_to_json
[rank0]:     sizes_match = all(size_equals_image(img) for img in seq)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 547, in <genexpr>
[rank0]:     sizes_match = all(size_equals_image(img) for img in seq)
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 544, in size_equals_image
[rank0]:     img_width, img_height = image.image.size  # type: ignore
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/wandb/sdk/data_types/image.py", line 689, in image
[rank0]:     self._image.load()
[rank0]:   File "/share/phoenix/nfs02/S6/localdisk/hj453/.conda/envs/neural-gaffer/lib/python3.9/site-packages/PIL/ImageFile.py", line 269, in load
[rank0]:     n, err_code = decoder.decode(b)
[rank0]: KeyboardInterrupt
